### PR TITLE
KubeCon NA 2023 Blog Post

### PR DESCRIPTION
**Pull request by svrnm**
TODO:
* Review list of talks
* Add location of project pavilion

### CODE REVIEW

File: content/en/blog/2023/kubecon-na.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+title: Join us for OpenTelemetry Talks and Activities at KubeCon NA 2023
+linkTitle: KubeCon NA '23
+date: 2023-09-11
+# prettier-ignore
+cSpell:ignore: Contribfest Pivotto Purvi Kanal Flamegraphs Sharone Zitzman Itiel Shwartz Komodor Ohly Shivanshu Shrivastava Kota Endo Masanori Itoh Danielson Benedikt Bongartz Mirabella Matej Coralogix Anusha Reddy Narapureddy Jaglowski observ Chronosphere Aronoff Broadbridge Hrabovcak Shishi
+author: '[Severin Neumann](https://github.com/svrnm)'
+---
+
+The OpenTelemetry project maintainers, members of the governance committee, and
+technical committee are excited to be at [KubeCon NA][] in Chicago from November
```
Comment (theletterf) [inline content/en/blog/2023/kubecon-na.md:11]: > excited

Just a pet peeve, but I noticed that most blog announcements have at least one or two instances of "excited". This is understandable, but perhaps might get a bit tiring for the dedicated follower to read? "Woof, OTel folks are always excited!" :-)

File: content/en/blog/2023/kubecon-na.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+You can help us improve the project by sharing your thoughts and feedback about
+your OpenTelemetry adoption, implementation, and usage! We also invite you to
+fill out our [community survey][]. We will create action items from your
+comments as appropriate. Check [#otel-user-research][] in CNCF's Slack instance
+for survey results and action item updates to come after KubeCon NA.
+
+Come join us to listen, learn, and get involved in OpenTelemetry.
+
+See you in Chicago!
```
Comment (theletterf) [inline content/en/blog/2023/kubecon-na.md:107]: Same feeling with exclamation marks.

**Review by theletterf — COMMENTED** [review]
Just some thoughts around excitement words and related punctuation—can be ignored.

File: content/en/blog/2023/kubecon-na.md (hunk 1 above)
Comment (svrnm) [reply content/en/blog/2023/kubecon-na.md:11]: 🤣 ... fair point, there are alternatives I guess

File: content/en/blog/2023/kubecon-na.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+We are happy to announce, that for the first time, you can join the
+OpenTelemetry maintainers in making OpenTelemetry better for everyone during the
+[OpenTelemetry Contribfest](https://sched.co/1JWS7), on Wednesday November 8,
+2023 from 2:30pm - 3:55pm. There will be several opportunities to contribute:
+
+- Join the Collector maintainers and help fix bugs, add new features, and add
+  new transformation functions.
+- Help the OTel JS maintainers update the instrumentation libraries and ensure
+  they are outputting semantic convention-compliant telemetry data.
```
Comment (mhausenblas) [inline content/en/blog/2023/kubecon-na.md:26]: Maybe want to link to https://opentelemetry.io/docs/specs/semconv/ for context?

**Review by mhausenblas — APPROVED** [review]
LGTM, one minor suggestion inline

File: content/en/blog/2023/kubecon-na.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+- **[Tutorial: Exploring the Power of Metrics Collection with OpenTelemetry on Kubernetes](https://sched.co/1R2pr)**<br>
+  by Pavol Loffay & Benedikt Bongartz, Red Hat; Anthony Mirabella, AWS; Matej
+  Gera, Coralogix; Anusha Reddy Narapureddy, Apple<br> Wednesday, November 8 •
+  11:00am - 12:30pm
+- **[Introducing Logs in OpenTelemetry: What It Unlocks and How to Use It](https://sched.co/1R2r4)**<br>
+  by Morgan McLean, Splunk & Dan Jaglowski, observIQ<br> Wednesday, November 8 •
+  2:30pm - 3:05pm
+- **[Jaeger the Future with OpenTelemetry and Metrics](https://sched.co/1R2q1)**<br>
+  by Pavol Loffay, Red Hat<br> Wednesday, November 8 • 3:25pm - 4:00pm
+- **[Tutorial: OpenTelemetry 101: Let's Instrument!](https://sched.co/1R2s9)**<br>
```
Comment (Aneurysm9) [inline content/en/blog/2023/kubecon-na.md:59]:
```suggestion
- **[Tutorial: OpenTelemetry 101: Let's Instrument!](https://sched.co/1R2sI)**<br>
```

**Comment by svrnm** [comment]
@mtwo @austinlparker please take a look, anything we can add around the loung / extra space for otel, cc @jpkrohling

File: content/en/blog/2023/kubecon-na.md (hunk 4 above)
Comment (svrnm) [reply content/en/blog/2023/kubecon-na.md:59]: thanks for catching that!

File: content/en/blog/2023/kubecon-na.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+technical committee are thrilled to be at [KubeCon NA][] in Chicago from
+November 6 - 9, 2023.
+
+Read on to learn about all the things related OpenTelemetry during KubeCon.
+
+## OpenTelemetry Contribfest
+
+We are happy to announce, that for the first time, you can join the
+OpenTelemetry maintainers in making OpenTelemetry better for everyone during the
+[OpenTelemetry Contribfest](https://sched.co/1JWS7), on Wednesday November 8,
```
[the remark this answers is not in the document]
Comment (svrnm) [reply content/en/blog/2023/kubecon-na.md:20]:
```suggestion
[OpenTelemetry Contribfest](https://sched.co/1R2rQ), on Wednesday November 8,
```

File: content/en/blog/2023/kubecon-na.md (hunk 3 above)
Comment (svrnm) [reply content/en/blog/2023/kubecon-na.md:26]:
```suggestion
  they are outputting [semantic convention-compliant](https://opentelemetry.io/docs/specs/semconv/) telemetry data.
```

File: content/en/blog/2023/kubecon-na.md (hunk 2 above)
Comment (svrnm) [reply content/en/blog/2023/kubecon-na.md:105]: I removed (most) other exclamation marks to keep that one ;)

**Comment by jpkrohling** [comment]
I would like to get @mtwo's approval before mentioning the lounge.

**Review by frzifus — APPROVED** [review]

**Review by cartermp — APPROVED** [review]
Overall LGTM

**Comment by mtwo** [comment]
+1 to Austin's comment, we should add a section about the OpenTelemetry Observatory area on the show floor, and we should direct community members to host sessions there

**Comment by svrnm** [comment]
> @svrnm I sent you a blurb on slack for the lounge, can you add it?

done, see https://github.com/open-telemetry/opentelemetry.io/pull/3251/commits/b06bb0e22926d6b926358312be5d54c8793de8d5

please take a look again if this fits

**Pull request merged**